src/interface/compare.jl

```diff
@@ -55,3 +55,69 @@ end
 function Base.isequal(A::AbstractArray, B::AbstractTensor)
     return helper_isequal(A, B)
 end
+
+function helper_argmin(A, dims)
+    @assert 1 <= dims <= length(size(A))
+    if (length(size(A)) >= 2)
+        return map(x -> x[2], reduce(minby, map(Pair, A, CartesianIndices(size(A))), dims=dims, init=Inf=>CartesianIndex(fill(0, length(size(A)))...)))
```

Comment:
When we define these high-level functions, the pattern is to first define the function for lazy arguments in lazy.jl: https://github.com/finch-tensor/Finch.jl/blob/05f4a3d66cf51af30569715025daeb5d95f4efa5/src/interface/lazy.jl#L526

then define the eager version in terms of the lazy one in eager.jl: https://github.com/finch-tensor/Finch.jl/blob/05f4a3d66cf51af30569715025daeb5d95f4efa5/src/interface/eager.jl#L170-L172